arch/arm/core/aarch64/reset.S

```diff
@@ -22,34 +21,27 @@ _ASM_FILE_PROLOGUE
 /*
  * Platform may do platform specific init at EL3.
  * The function implementation must preserve callee saved registers as per
- * Aarch64 ABI PCS.
+ * AArch64 ABI PCS.
  */
+
 WTEXT(z_arch_el3_plat_init)
 SECTION_FUNC(TEXT,z_arch_el3_plat_init)
 ret
 
-/**
- *
- * @brief Reset vector
+/*
+ * Reset vector
  *
  * Ran when the system comes out of reset. The processor is in thread mode with
  * privileged level. At this point, neither SP_EL0 nor SP_ELx point to a valid
  * area in SRAM.
  *
  * When these steps are completed, jump to z_arm64_prep_c(), which will finish
  * setting up the system for running C code.
- *
- * @return N/A
  */
+
 GTEXT(__reset)
 SECTION_SUBSEC_FUNC(TEXT,_reset_section,__reset)
 
-/*
- * The entry point is located at the __reset symbol, which is fetched by a XIP
- * image playing the role of a bootloader, which jumps to it, not through the
```

Comment:
are this and other multi-line comments that were removed in this patch no longer correct?
when it comes to assembly files I often think it's worth being verbose when explaining things.

Reply:
That was a leftover from the old Cortex-M file that I used as reference at the beginning of the port. XIP image is not currently supported and I'm not even sure that it works on AArch64 the same way it does on the ARMv7. So I just decided to take that out.